modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java

```diff
@@ -1013,6 +1013,26 @@ public void run() {
     StatusThreadPoolExecutor.getInstance().execute(r);
   }
 
+  @ReactMethod
+  public void getContactCode(final String identity, final Callback callback) {
```

Comment:
hmm, why is it only for Android?

Reply:
good, point, totally forgot about desktop and ios, thanks!